psydac/feec/tests/test_global_projectors.py

```diff
@@ -92,15 +101,17 @@ def test_L2_projector_1d(domain, ncells, degree, periodic, nquads):
 @pytest.mark.parametrize('ncells', [[200,200]])
 @pytest.mark.parametrize('degree', [[2,2], [2,3], [3,3]])
 @pytest.mark.parametrize('periodic', [[False, False], [True, True]])
+@pytest.mark.parametrize('multiplicity', [[1,1],[2,2]])
 
-def test_derham_projector_2d_hdiv(ncells, degree, periodic):
+def test_derham_projector_2d_hdiv(ncells, degree, periodic, multiplicity):
 
     domain = Square('Omega', bounds1 = (0,2*np.pi), bounds2 = (0,2*np.pi))
     domain_h = discretize(domain, ncells=ncells, periodic=periodic)
-    
+    #change mulitplicity if higher than degree to avoid problems (case p<m doesn't work)
+    multiplicity = [min(m,p) for p, m in zip (degree, multiplicity)]
```

Comment:
Typo: mulitplicity -> multiplicity

More readable:
```suggestion
    #change multiplicity if higher than degree to avoid problems (case p<m doesn't work)
    multiplicity = [min(m, p) for p, m in zip(degree, multiplicity)]

```